modules/recognize-resource-limits-quotas.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+// * web_console/web-console.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="recognize-resource-limits-quotas"]
+= Recognizing resource and project limits and quotas
+
+You can view a graphical representation of available resources in the *Topology* view of the web console *Developer* perspective.
+
+If a resource has a message about resource limitations or quotas being reached, a yellow border appears around the resource name. Click the resource to open a side panel to see the message. If the *Topology* view has been zoomed out, a yellow dot indicates that a message is available.
+
+If using *List View* from the *View Shortcuts* menu, resources appear as a list. The *Alerts* column indicates if a message is available.
+
```

Comment:
*PEER REVIEW* I created a new module from [existing text](https://github.com/openshift/openshift-docs/blob/main/modules/customizing-the-web-console-URL.adoc#customizing-the-console-route), without change.